Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ScreenData.purs

```diff
@@ -65,6 +65,14 @@ initData = {
     , messages : []
     , suggestionsList : []
     , messageToBeSent : ""
+    , bannerViewState : {
+        backgroundColor : "#F0FAF0"
```

Comment:
@aryanguptaSG put these colors inside Colors.purs and use it by importing from the same.